docs/source/models/causes/opioid_use_disorder/index.rst

```diff
@@ -0,0 +1,821 @@
+
+3. **No Differential Opioid Types**: The model does not distinguish between different opioid types (prescription vs. illicit; heroin vs. fentanyl) or routes of administration (oral vs. injection), which have different risk profiles.
+
+4. **Treatment Effectiveness**: The model assumes that MOUD substantially reduces disability and mortality, potentially to zero in the base case. Actual treatment effectiveness varies by medication type, dosage, adherence, and individual factors.
+
+5. **Steady-State Assumption**: Parameter estimation using DisMod-AT assumes approximate steady state or smooth temporal trends. Rapid changes in the opioid epidemic (e.g., fentanyl emergence) may violate this assumption.
+
+6. **Natural Remission**: The natural remission rate (*r*) represents spontaneous recovery without formal treatment. This is poorly characterized in the literature and is primarily identified through model calibration to match observed prevalence.
+
+**Limitations**
+
+1. **Treatment Coverage Data**: Estimates of the proportion of individuals with OUD receiving MOUD are highly uncertain and vary considerably across settings, populations, and data sources.
+
+2. **Remission Measurement**: Natural remission from OUD is difficult to measure directly, as individuals who recover without treatment are unlikely to be captured in treatment or surveillance systems.
+
+3. **Relapse Dynamics**: The model does not explicitly capture multiple cycles of treatment and relapse, which are common in OUD. Individuals may transition multiple times between states.
+
+4. **Comorbidities**: The model does not explicitly represent common comorbidities (mental health disorders, polysubstance use, infectious diseases) that influence OUD progression and treatment outcomes.
+
+5. **Heterogeneity**: Important sources of heterogeneity (age of onset, severity, social determinants, genetic factors) are not explicitly represented in the basic three-state model.
+
+6. **Overdose Mortality**: While the model includes excess mortality associated with OUD, overdose deaths may be coded separately in mortality data and require careful reconciliation with OUD-attributed deaths.
+
+7. **Intervention Effects**: Estimating the effect of specific interventions on transition rates (e.g., how much does a particular treatment expansion program increase *ti*?) requires additional assumptions and calibration.
+
+**Implications for Interpretation**
+
+Results from this model should be interpreted as population-level estimates capturing average dynamics across heterogeneous individuals. Scenario analyses and interventions should be evaluated based on their effects on transition rates, with careful attention to the assumptions embedded in those rate changes. Uncertainty in parameter estimates (particularly treatment-related transitions) should be thoroughly characterized and propagated through model outputs.
```

Comment:
I don't think this adds much. In particular, the first sentence is kind of incompatible with our microsimulation approach. Interventions are not yet described. Uncertainty is described on this page.